apps/server/src/schema/feedback/feedback-helper.ts

```diff
@@ -0,0 +1,29 @@
+import { type Feedback } from '@repo/database';
+import { logger } from '@repo/logger';
+import { Environment, MODE } from '@repo/mode';
+import { AError } from '@repo/utils';
+
+export const postFeedbackToSlack = async (feedback: Feedback): Promise<AError | undefined> => {
+  const slackMessage = {
+    text: `New ${MODE !== Environment.Production ? 'Test ' : ''} Feedback Received: \n*Type:* ${feedback.type} \n*Message:* ${feedback.message}`,
+  };
+
+  const slackWebhookUrl = process.env.SLACK_WEBHOOK_URL_PUBLIC_FEEDBACK;
```

Comment:
```suggestion
  const slackWebhookUrl = env.SLACK_WEBHOOK_URL_PUBLIC_FEEDBACK;
```
This is how you refer it now that you put it in config. Now it is typed as well